README.md

```diff
@@ -77,28 +96,37 @@ Users are individuals seeking a communication platform. They range from casual u
 ### 3. Specific Requirements
 
 #### 3.1 User Requirements
--   Users must be able to register, login, and manage their profiles.
+-   Users should be able to register for a new account.
+-   Users should be able to log in to an existing account with their username and passcode.
+-   Users should be able to change their username, email, display name and avatar.
 -   Users must be able to add and remove friends.
--   Users must be able to create and manage groups
--   Users must be able to send, receive, edit, and delete messages.
--   Moderators in groups must be able to manage member roles and permissions.
+-   Users should be able to accept/deny friend requests from other users
+-   Users should be able to remove existing friends
+-   Users should be able to block other users
+-   Users should be able to create individual or group chats with other users
```

Comment:
user should be able to create direct messages with one of their friends, and create group chats to which they can invite more than one friend

Reply:
ok I guess, I mean Zahra and I both thought it looked pretty clear the way it is but if you feel that strongly about it, sure.

Reply:
since friendships & groups correspond to different functional requirements, it's more appropriate to treat them that way in the user requirements as well. looks good